src/git.ts

```diff
@@ -27,7 +27,7 @@ export const git = (log: Logger, octokit: Pick<OctokitInstance, 'pulls' | 'repos
       data: { files },
     } = commit
 
-    const filenames = files?.map(f => f.filename) ?? []
+    const filenames = files?.filter(f => f.status !== 'removed')?.map(f => f.filename) ?? []
```

Comment:
(files || [])
  .filter(f => f.status !== 'removed')
  .map(f => f.filename)
```suggestion
    const filenames = files?.filter(f => f.status !== 'removed')?.map(f => f.filename) ?? []
```